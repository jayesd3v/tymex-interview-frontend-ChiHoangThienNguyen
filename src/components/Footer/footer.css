footer {
    background: #16171A;
    color: #fff;
    font-size: 0.8em;
}

footer .title {
    font-weight: bold;
}

footer a{
    color: #fff;
    text-decoration: none;
    margin: 0 5px;
}