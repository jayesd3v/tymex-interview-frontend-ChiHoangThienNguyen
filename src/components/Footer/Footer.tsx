/* eslint jsx-a11y/anchor-is-valid: 0 */

import { Button, Col, Container, Form, Row } from 'react-bootstrap';
import './footer.css';

const Footer = () => {
    return (
        <footer className='pb-5 pt-5'>
            <Container>
                <Row>
                    <Col xs={12} md={5} lg={4}>
                        <div className='section mb-5'>
                            <div className='title mb-4'>NAVIGATION</div>
                            <Row>
                                <Col xs={4}>
                                    <ul className='list-unstyled'>
                                        <li>
                                            <a href='#'>Home</a>
                                        </li>
                                        <li>
                                            <a href='#'>About Us</a>
                                        </li>
                                        <li>
                                            <a href='#'>Our teams</a>
                                        </li>
                                    </ul>
                                </Col>
                                <Col xs={4}>
                                    <ul className='list-unstyled'>
                                        <li>
                                            <a href='#'>Home</a>
                                        </li>
                                        <li>
                                            <a href='#'>About Us</a>
                                        </li>
                                        <li>
                                            <a href='#'>Our teams</a>
                                        </li>
                                    </ul>
                                </Col>
                                <Col xs={4}>
                                    <ul className='list-unstyled'>
                                        <li>
                                            <a href='#'>Home</a>
                                        </li>
                                        <li>
                                            <a href='#'>About Us</a>
                                        </li>
                                        <li>
                                            <a href='#'>Our teams</a>
                                        </li>
                                    </ul>
                                </Col>
                            </Row>
                        </div>
                    </Col>
                    <Col xs={12} md={{ span: 5, offset: 1 }} lg={{ span: 4, offset: 0 }}>
                        <div className='section mb-5'>
                            <div className='title mb-4'>CONTACT US</div>
                            <ul className='list-unstyled'>
                                <li>
                                    <a href='#'>
                                        <img src='/phone.png' alt='phone' /> 012345678910
                                    </a>
                                </li>
                                <li>
                                    <a href='#'>
                                        <img src='/mail.png' alt='email' /> tymex-talent@tyme.com
                                    </a>
                                </li>
                            </ul>
                        </div>
                    </Col>
                    <Col xs={12} lg={{ span: 4, offset: 0 }}>
                        <div className='section mb-5'>
                            <div className='title mb-4'>SUBSCRIBE TO RECEIVE OUR LATEST UPDATE</div>
                            <Form>
                                <Form.Group as={Row}>
                                    <Col xs={12} sm={8}>
                                        <Form.Control placeholder='email@example.com' className='mb-3' />
                                    </Col>
                                    <Col xs={12} sm={4}>
                                        <Button>Subscribe</Button>
                                    </Col>
                                </Form.Group>
                            </Form>
                        </div>
                    </Col>
                </Row>
                <hr />
                <Row className='d-flex'>
                    <Col xs={12} md={6} className='text-center text-md-start'>
                        &copy; 2023 - Edit, All Rights Reserved.
                    </Col>
                    <Col xs={12} md={6} className='justify-content-center justify-content-md-end d-flex'>
                        <ul className='list-unstyled list-group list-group-horizontal text-white                                                                                                                                                                                    '>
                            <li>
                                <a href='#'>Security</a>
                            </li>
                            <li>
                                <a href='#'>Legal</a>
                            </li>
                            <li>
                                <a href='#'>Privacy</a>
                            </li>
                        </ul>
                    </Col>
                </Row>
            </Container>
        </footer>
    );
};

export default Footer;
