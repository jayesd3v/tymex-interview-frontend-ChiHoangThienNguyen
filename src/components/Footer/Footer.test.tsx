import { render } from "@testing-library/react";
import Footer from "./Footer";

describe('Footer tests', () => {
    test('snapshot test', () => {
        const { container } = render(<Footer />);
        expect(container).toMatchSnapshot();
    });
});
