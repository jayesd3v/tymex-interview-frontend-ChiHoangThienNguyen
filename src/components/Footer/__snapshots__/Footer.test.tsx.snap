// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests snapshot test 1`] = `
<div>
  <footer
    class="pb-5 pt-5"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-4 col-md-5 col-12"
        >
          <div
            class="section mb-5"
          >
            <div
              class="title mb-4"
            >
              NAVIGATION
            </div>
            <div
              class="row"
            >
              <div
                class="col-4"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Home
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      About Us
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Our teams
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="col-4"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Home
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      About Us
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Our teams
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="col-4"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Home
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      About Us
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Our teams
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-lg-4 col-md-5 col-12 offset-lg-0 offset-md-1"
        >
          <div
            class="section mb-5"
          >
            <div
              class="title mb-4"
            >
              CONTACT US
            </div>
            <ul
              class="list-unstyled"
            >
              <li>
                <a
                  href="#"
                >
                  <img
                    alt="phone"
                    src="/phone.png"
                  />
                   012345678910
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <img
                    alt="email"
                    src="/mail.png"
                  />
                   tymex-talent@tyme.com
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          class="col-lg-4 col-12 offset-lg-0"
        >
          <div
            class="section mb-5"
          >
            <div
              class="title mb-4"
            >
              SUBSCRIBE TO RECEIVE OUR LATEST UPDATE
            </div>
            <form
              class=""
            >
              <div
                class="row"
              >
                <div
                  class="col-sm-8 col-12"
                >
                  <input
                    class="mb-3 form-control"
                    placeholder="email@example.com"
                  />
                </div>
                <div
                  class="col-sm-4 col-12"
                >
                  <button
                    class="btn btn-primary"
                    type="button"
                  >
                    Subscribe
                  </button>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
      <hr />
      <div
        class="d-flex row"
      >
        <div
          class="text-center text-md-start col-md-6 col-12"
        >
          Â© 2023 - Edit, All Rights Reserved.
        </div>
        <div
          class="justify-content-center justify-content-md-end d-flex col-md-6 col-12"
        >
          <ul
            class="list-unstyled list-group list-group-horizontal text-white                                                                                                                                                                                    "
          >
            <li>
              <a
                href="#"
              >
                Security
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Legal
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Privacy
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
